lib/ParseSIL/ParseSIL.cpp

```diff
@@ -6742,6 +6745,242 @@ bool SILParserTUState::parseSILDefaultWitnessTable(Parser &P) {
   return false;
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+// TODO(TF-893): Dedupe with `SILParser::convertRequirements` upstream.
```

Comment:
Could you please add a comment to explain why this duplication is necessary?

Reply:
Done in https://github.com/apple/swift/pull/27487/commits/2073f863826fb305c1fc105a520d84de1e1463ec.

`convertRequirements` is currently defined on `SILParser`, but it needs to be moved to `Parser` for differentiability witness parsing.